maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,18 @@
+.nav {
+  align-items: center;
+  background-color: white;
+  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
```

Comment:
this shadow needs to be more visible and equal looking as possible to the image attached to the card

Reply:
new box-shadow.

![image](https://user-images.githubusercontent.com/21252506/59781766-1b200d80-9282-11e9-830d-aa38eaf92aa5.png)